src/cljs/rems/table.cljs

```diff
@@ -18,16 +18,21 @@
 (defn column-class [column-definitions col]
   (get-in column-definitions [col :class] (name col)))
 
+(defn column-class-customized [item column-definitions col]
```

Comment:
How about introducing to the table component, an optional `:row-class` callback function that you give the item to that can then give back a class to be applied to the `tr` element?